### DISCUSSION TITLE
v-0.0.0.0.0.0.0.0.1

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#35] [Announcements]
We'll be counting down from zero with release numbers for now.

New this release!
* `delete` keyword by @gaearon 
* low-level memory management by @scolsen 
* variable overloading by @scolsen 
* dividing by zero now returns `undefined`
* notes for copilot users

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/v-0.0.0.0.0.0.0.0.1'>v-0.0.0.0.0.0.0.0.1</a>.</em>

**Comment by ridho-aru** [comment]
I'm just following Dan, so...
